catroid/src/main/java/org/catrobat/catroid/stage/StageLifeCycleController.java

```diff
@@ -221,6 +224,7 @@ public static void stageResume(final StageActivity stageActivity) {
 
 	static void stageDestroy(StageActivity stageActivity) {
 		if (checkPermission(stageActivity, getProjectsRuntimePermissionList())) {
+			deleteLookFilesToBeDeleted();
```

Comment:
This makes me think, is there a test that guarantees that the look was deleted? :thinking: 
If not please add one (Unit Test)